files/resources/nodenet-w-fip.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-heat_template_version: pike
+heat_template_version: queens
```

Comment:
It's queens here because prior to Queens the allowed_address_pairs attribute required a MAC address as well (apparently)